optimism.tokenlist.json

```diff
@@ -1199,11 +1199,33 @@
       "extensions": {
         "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
       }
+    },
+    {
+      "chainId": 1,
+      "address": "0x03e173ad8d1581a4802d3b532ace27a62c5b81dc",
+      "name": "Thales",
+      "symbol": "THALES",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/THALES.svg",
+      "extensions": {
+        "optimismBridgeAddress": "0x99C9fc46f92E8a1c0deC1b1747d010903E884bE1"
+      }
+    },
+    {
+      "chainId": 10,
+      "address": "0x217D47011b23BB961eB6D93cA9945B7501a5BB11",
```

Comment:
This token is pointing to a different L1 Thales token at `0x8947da500eb47f82df21143d0c01a29862a8c3c5` not the `0x03e173ad8d1581a4802d3b532ace27a62c5b81dc` address above. Which is the correct L1 token you want to bridge? And is there a reason why you deployed a dedicated Optimism Thales token on L1?

Reply:
Also note that you can use the token factory described [here](https://community.optimism.io/docs/guides/token-dev/#getting-some-testnet-eth) to create your L2 token.